e2e/deploy-vault.go

```diff
@@ -20,6 +20,7 @@ var (
 	vaultConfigPath      = "kms-config.yaml"
 	vaultTenantPath      = "tenant-sa.yaml"
 	vaultTenantAdminPath = "tenant-sa-admin.yaml"
+	vaultUserSecretPath  = "user-secret.yaml"
```

Comment:
`vaultUserSecretPath` should be renamed `vaultUserSecret` as it is not a path, rather a file.

I understand that this is the convention throughout the file, but it is never too late to improve ;) 
Whether to update it for your changes, or do it for the other variables too, I leave that choice to you.